parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -116,8 +116,12 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
+	Migrations,
 >;
 
+pub type Migrations =
+	(pallet_balances::migration::MigrateToTrackInactive<Runtime, xcm_config::CheckingAccount>,);
```

Comment:
Ah, sorry for the confusion in chat. You can leave this out since it will be from genesis. It will initialize to zero and the Balances/XCM logic will keep the inactive data up to date.

Reply:
ok, removed, thank you